### PR TITLE
Add pipeline stage survey_auto_code, which exports wt_demog_1 to Coda

### PR DESCRIPTION
**Pull request by as2388**
Hopefully this is clear from the TODOs, but the intention here is to extend this stage to merge in the other demographic survey and clean that here too in future.

### CODE REVIEW

File: survey_auto_code/survey_auto_code.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    # TODO: Extend to add another argument for demog2, load this survey, and merge with demog1.
+    # TODO: Then we don't have to maintain parallel pipelines for data which was arbitrarily separated to begin with.
+
+    # Set missing entries in the raw data to 'NA'
+    for td in data:
+        for key in demog_keys:
+            long_key = "{} (Text) - wt_demog_1".format(key)
+            if long_key not in td:
+                td.append_data({long_key: "NA"}, Metadata(user, Metadata.get_call_location(), time.time()))
```
Comment (lukechurch) [inline survey_auto_code/survey_auto_code.py]: Please can we extract "NA" into a global constant

File: survey_auto_code/survey_auto_code.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    # Output for manual verification + coding
+    if not os.path.exists(coded_output_path):
+        os.makedirs(coded_output_path)
+
+    for key in demog_keys:
+        output_file_path = path.join(coded_output_path, "{}.csv".format(key))
+        with open(output_file_path, "w") as f:
+            TracedDataCodaIO.export_traced_data_iterable_to_coda_with_scheme(
+                data, "{} (Text) - wt_demog_1".format(key), "{}_clean".format(key), key, f)
```
Comment (lukechurch) [inline survey_auto_code/survey_auto_code.py]: We seem to be constructing this magic string twice, I don't mind too much, but if we're going to be doing this often, or not clearly near each other, could we factor out the string constant?

File: run_scripts/05_06_survey_auto_code.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+USER=$1
+DATA_ROOT=$2
+
+cd ../survey_auto_code
+
+mkdir -p "$DATA_ROOT/05 Auto-Coded"
+mkdir -p "$DATA_ROOT/06 Coda Files"
+
+# Auto-code wt_demog_1 and export to Coda. TODO: Auto-code the other surveys and export to Coda.
```
Comment (lukechurch) [inline run_scripts/05_06_survey_auto_code.sh]: Please could you put the TODO on it's own line so it's easier to see whilst skim reading?

File: survey_auto_code/survey_auto_code.py (hunk 1 above)
Comment (as2388) [reply survey_auto_code/survey_auto_code.py]: Will fix once this (https://github.com/AfricasVoices/CoreDataModules/pull/34) is merged

File: survey_auto_code/survey_auto_code.py (hunk 2 above)
Comment (as2388) [reply survey_auto_code/survey_auto_code.py]: This will need improving anyway to cope with demog_2. Will fix then.

**Pull request merged**